vignettes/faq.Rmd

```diff
@@ -49,8 +49,8 @@ knitr::opts_chunk$set(
  
 ##### Why do we use a **certain R version and package versions** for development?
 
-* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We currently work in R Version 3.6.3, but that will change as we move forward with `{admiral}`. This need for a common development environment also carries over for our choice of package versions.  
-* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against R Version 3.6.3 with a CRAN package snapshot from 2020-02-29, R Version 4.0 with a CRAN package snapshot from 2021-03-31 and the latest R version with the latest snapshots of packages.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
+* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We only use the three latest R versions and the closest date of snapshots of R packages available when that R version came out. This need for a common development environment also carries over for our choice of package versions.  
```

Comment:
This paragraph refers to our dev environment which is only a single R version. We should only reference the 3 version we test on in the paragraph below.

Reply:
Sorry he is saying that as developers we only work in 1 R version.  The earliest of the 3 latest R versions.  So 4.0 is earliest of 4.0, 4.1 and 4.2

Reply:
We test against 3 R versions